package.json

```diff
@@ -18,7 +18,12 @@
   },
   "dependencies": {
     "autoprefixer": "^10.2.1",
+
```

Comment:
Please revert the file back to the previous state.
Axios is imported 2 times here.